{
  "contest": 116,
  "handle": "sorrynotsorry",
  "address": "0x6262f720679C04cc7f2a14c0c03486D0287c4853",
  "risk": "2",
  "title": "Calls inside loops that may address DoS.",
  "issueId": 126,
  "issueUrl": "https://github.com/code-423n4/2022-05-cudos-findings/issues/126"
}